wiki/grips-and-tricks.md

```diff
@@ -19,16 +19,37 @@ Additionally, here's an image from the game developers about how the
 
 ![Bloq Hitbox](~@images/mapping/hitbox-from-split.jpg)
 
-## Ranking
+## Ranking Systems
 *How does global ranking work? What are performance points? What are the numbers in my brackets? Why is my PP so small?*
 
-You can find your global ranking and PP on [ScoreSaber](https://scoresaber.com/global) - here you can search for your name.
+There are 2 separate ranking systems: [ScoreSaber](https://scoresaber.com/global) and [BeatLeader](https://www.beatleader.xyz/ranking/1).
+You can find your global ranking and PP on their respective sites where you can search for your name.
+
+On **PC**, your Beatleader account is tied to your Steam account and is your steam name by default. <br>
```

Comment:
What happens if someone uses the oculus version of the game?

Reply:
they'll need to make a BL account or connect it to their steam account. Added